oidc-fapi-conformance-tests/export_results_fapi.py

```
@@ -17,35 +17,49 @@ def save_results(plan, i):
-        'warnings': test_warnings,
-        'others': test_others
+        'errors': test_errors
     }
 
 
 if __name__ == '__main__':
     failed_plan_details = dict()
     contains_fails = False
     warnings.filterwarnings("ignore")
-    plan_list = json.loads(requests.get(url=conformance_suite_url + "/api/plan?length=50", verify=False).content)
+    plan_response = requests.get(url=conformance_suite_url + "/api/plan?length=50", verify=False)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)